### ISSUE TITLE
[MCU] Create / modify existing project to demonstrate module functionality

### ISSUE THREAD

**Issue by AndreasFreund** [amosproj/amos2024ss07-updating-flash-boot-loader#25]
## User story
1. As a developer
2. I need there to be only a single project for Aurix IDE
3. So that it can run on the MCU

## Acceptance criteria
* only a single aurix project left in the repository
* directory structure that keeps hardware driver modules seperate from the logic and seperate from the mcu (e.g. core0.c)
* single project should demonstrate all features of the modules
* watch out for flash storage wearing out (add a delay before demonstrating the flash functionality, only do one action and don't erase the flash in the while loop)
* The file that demonstrates the modules should be seperate from the main file and not include any of the aurix headers
* This file will become the logic of the bootloader later on

Note: Some of these tasks are already half done

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by ibexq** [comment]
@sebas-rodri will review
